drivers/watchdog/wdt_nrfx.c

```diff
@@ -53,6 +54,7 @@ static int wdt_nrf_setup(const struct device *dev, uint8_t options)
 
 	nrfx_wdt_enable(&config->wdt);
 
+	data->enabled = true;
```

Comment:
You also need to add `data->enabled = false;` in `wdt_nrf_disable()`.

Reply:
done